targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommonSettings.cpp.ejs

```diff
@@ -0,0 +1,30 @@
+<%- copyright %>
+
+#include "PlayFabCommonSettings.h"
+
+namespace PlayFabCommon
+{
+    const FString PlayFabCommonSettings::sdkVersion = "<%- sdkVersion %>";
```

Comment:
[ASK] These do not get populated correctly. They just show up as empty. Any idea?
Have already added unrealmarketplaceplugin to the SDKManualNotes.json file in the API_SPECS repo in my local branch.

Reply:
fixed